src/main/java/StorageInterface.java

```diff
@@ -0,0 +1,3 @@
+interface StorageInterface {
```

Comment:
Not sure what purpose `StorageInterface` serves, might want to add some functions to it!